.github/workflows/build.yml

```diff
@@ -191,28 +191,6 @@ jobs:
       -   run: yarn test:e2e:${{ matrix.backend }}:${{ matrix.mode }}
           shell: bash
 
-  e2e-sync:
```

Comment:
Starting from 1.12.0 geth doesn't support ethash and stops with : "`Fatal: Failed to register the Ethereum service: ethash is only supported as a historical component of already merged networks`" so sync test scripts are removed

Reply:
OMG. do we need somehow to adapt our sync test? Do we need to create an issue for this?

Reply:
As Eth mainnet is on POS so we dnt need to invoke sync tests, but let these be in tests so any one using web3.js on other NWs with POW can manually invoke these tests.